data/src/main/java/com/teamagam/gimelgimel/data/response/entity/UserLocationResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.teamagam.gimelgimel.data.response.entity;
+
+import com.teamagam.gimelgimel.data.response.entity.contents.LocationSampleData;
+import com.teamagam.gimelgimel.data.response.entity.visitor.IResponseVisitor;
+
+/**
+ * UserLocation-Type class for {@link GGResponse}'s inner content
+ */
+public class UserLocationResponse extends GGResponse<LocationSampleData> {
```

Comment:
This is a true UserLocationResponse because it will be converted only into a UserLocation object.